test/gtest/ucp/test_ucp_peer_failure.cc

```diff
@@ -360,13 +360,21 @@ void test_ucp_peer_failure::do_test(size_t msg_size, int pre_msg_count,
         EXPECT_NE(UCS_OK, m_err_status);
 
         if (UCS_PTR_IS_PTR(sreq)) {
-            /* The request may either succeed or fail, even though the data is
+            ucs_status_t status;
+            /* TODO update comment
```

Comment:
1. why needed? if m_err_count!=0, i would expect the request to also complete
2. update comment
3. can we use request_wait()?

Reply:
discussed offline:
1. use request_wait
2. update comment: "If rendezvous protocol is used, the m_err_count is increased on the receiver side, so the send request may not complete immediately"